File: js210/js210_small_problems/easy_5/counting_up.js
"use strict"

function sequence(integer) {
  let integers = [];

  for (let currentInt = 1; currentInt <= integer; currentInt += 1) {
    integers.push(currentInt);
  }

  return integers;
}

console.log(sequence(5));
console.log(sequence(3));
console.log(sequence(1));

File: rb109/Practice/Easy 9/counting_up.rb
# counting_up.rb

# write a method
# input: integer
# output: array of all integers, in sequence between 1 and the argument

def sequence(number)
  (1..number).to_a
end

p sequence(5) == [1, 2, 3, 4, 5]
p sequence(3) == [1, 2, 3]
p sequence(1) == [1]
